### DISCUSSION TITLE
Embed images as base64

### DISCUSSION THREAD

**Discussion by Haaxor1689** [maizzle/framework#1023] [Q&A] [answered]
I have just some smaller icons in my email template and was wondering if it was worth it trying to inline them as base64 to get around some email client's limitations around not loading external images. Any one tried automating this or probably is it not a good idea at all?

**Comment by cossssmin** [comment, marked as answer]
Depends what email clients you want to support, for example Gmail and Outlook don't support base64:

https://www.caniemail.com/features/image-base64/
